Gems/Atom/Feature/Common/Assets/Materials/Types/Debug/DebugVertexStreams.azsl

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
We might consider putting this in Gems\Atom\Feature\Common\Assets\Materials\Special.
That folder is intended for utility material types that often only have a single material, or are used by the system in some way. I think DebugVertexStreams fits that.